src/controllers/dlgprefcontrollers.cpp

```diff
@@ -110,19 +110,6 @@ void DlgPrefControllers::setupControllerWidgets() {
     qSort(controllerList.begin(), controllerList.end(), controllerCompare);
```

Comment:
GitHub won't let me comment at the correct place because you haven't edited this code, but adding
```
    if (clickedItem == nullptr) {
        return false;
    }
```
to the beginning of DlgPrefControllers::handleTreeItemClick fixes the segfault on shutdown.

Reply:
I've fixed the crash in another way. Can you confirm that we are talking about the same crash and that it is fixed now?